experiments/ClimaEarth/components/ocean/clima_seaice.jl

```diff
@@ -0,0 +1,521 @@
+import Oceananigans as OC
+import ClimaSeaIce as CSI
+using ClimaSeaIce.SeaIceThermodynamics.HeatBoundaryConditions:
+    IceWaterThermalEquilibrium, MeltingConstrainedFluxBalance, get_tracer
+import ClimaOcean as CO
+import ClimaCoupler: Checkpointer, FieldExchanger, FluxCalculator, Interfacer, Utilities
+import ClimaComms
+import ClimaCore as CC
+import Thermodynamics as TD
+import ClimaOcean.EN4: download_dataset
+using KernelAbstractions: @kernel, @index, @inbounds
+using RootSolvers: find_zero, SecantMethod, CompactSolution
+
+include("climaocean_helpers.jl")
+
+# Rename ECCO password env variable to match ClimaOcean.jl
+haskey(ENV, "ECCO_PASSWORD") && (ENV["ECCO_WEBDAV_PASSWORD"] = ENV["ECCO_PASSWORD"])
+
+"""
+    ClimaSeaIceSimulation{SIM, A, REMAP, NT, IP}
+
+The ClimaCoupler simulation object used to run with ClimaSeaIce.
+This type is used by the coupler to indicate that this simulation
+is an surface/ocean simulation for dispatch.
+
+It contains the following objects:
+- `ice::SIM`: The ClimaSeaIce simulation object.
+- `area_fraction::A`: A ClimaCore Field representing the surface area fraction of this component model on the exchange grid.
+- `remapping::REMAP`: Objects needed to remap from the exchange (spectral) grid to Oceananigans spaces.
+- `ocean_ice_fluxes::NT`: A NamedTuple of fluxes between the ocean and sea ice, computed at each coupling step.
+- `ice_properties::IP`: A NamedTuple of sea ice properties, including melting speed, Stefan-Boltzmann constant,
+    and the Celsius to Kelvin conversion constant.
+"""
+struct ClimaSeaIceSimulation{SIM, A, REMAP, NT, IP} <: Interfacer.SeaIceModelSimulation
+    ice::SIM
+    area_fraction::A
+    remapping::REMAP
+    ocean_ice_fluxes::NT
+    ice_properties::IP
+end
+
+"""
+    ClimaSeaIceSimulation()
+
+Creates an ClimaSeaIceSimulation object containing a model, an integrator, and
+a surface area fraction field.
+This type is used to indicate that this simulation is an ocean simulation for
+dispatch in coupling.
+
+Initially, no sea ice is present.
```

Comment:
This is not the case if a start_date is provided? (and we always have a start_date for CMIP / AMIP?)

Reply:
oops, the docstring isn't updated. I'll fix it